### ISSUE TITLE
Hydrostate doesnt hold engine speed when in first mode

### ISSUE THREAD

**Issue by Janlul05** [s4t4n/FS22_CVT_Addon#40]
So what i am trying to say is that if i I drive a combine with the first mode selected and i drive with hand throttle then the engine speed always reduces to like 1600 rpm. But that shouldn't be the case if i am on full throttle. I also use the keep speed feature from the VCA mod if that’s necessary. I have driven combines in real live and when i give it full throttle it stays there cause you don't control the speed with throttle not directly at least. SO i don’t understand why it’s different in the simulator. I am sorry if my English isn’t the best I am from Austria.

but otherwise thanks for your hard work, just wanted to check if you can do something about it.

best regards.

**Comment by atledreier** [comment]
Just adding to this. I drive a JD6430, and that has a hydrostatic (ot at least one that acts JUST like the Hydrostatic in this addon). I can confirm it stays at or near idle with the gearbox in low or "1" in the game. This is not correct. The real tractor sits at around 1400rpm when driving in the "low" setting and little load, and will increase to near redline under heavy load and acceleration. In high, or "2" it behaves much more as expected.

The engine braking effect is very pronounced with throttle position changes too, much more than the real thing. The real thing is nice and smooth, and will only gradually bleed off speed when you release the throttle, you actually use the brake quite a lot. It will also creep forward at idle, not stop completely as in the sim currently, so there might be a way to simulate this better within the game engine limitations?

**Comment by atledreier** [comment]
I checked, it takes about 4 seconds from 18-19kph to 5-6kph where the deceleration tapers off to a creep.
This is in the "low" range on the JD 6430.

**Comment by atledreier** [comment]
https://github.com/s4t4n/FS22_CVT_Addon/assets/54736373/ccad9270-9f05-4044-8699-272e245c08c7

**Comment by atledreier** [comment]
Another followup, and I see that mode 1 works as intended, i just hadn't RTFM...  *facepalm*
I falsely assumed this would be equivalent to the "low" and "high" ranges of the gearbox, not actual operating modes. Sorry about that.

But the engine brake issue in mode 2 is still relevant, so maybe have a look into that?

**Comment by s4t4n** [comment]
hehe, but nice for the RL Video.. I would love if you can do more driving situations... engine/gearbox brake, acceleration with weight or in work with a culviator or something.
The rpm ranges etc.

the pto is enabled because of the ipm to use? :D

**Comment by atledreier** [comment]
For the intents of the game and this mod, it's the mode that by far comes closest to the actual thing when actually driving it, so that's why I chose HST mode 2.  :-)  I'll dig around a bit.

Now in winter there isn't really much heavy use, but we've got some hills and hauling watertanks loads the tractor some, so I'll try to catch some more situations, like getting going from a standstill with a heavy load, transitioning from flat to uphill with load, and from flat do downhill engine braking.
